frontend/packages/kubevirt-plugin/integration-tests-cypress/view/selector.ts

```diff
@@ -66,3 +70,11 @@ export enum storageClass {
 
 // yaml page
 export const createYAMLButton = 'button[data-test="save-changes"]';
+
+export const kebabBtn = 'button[data-test-id="kebab-button"]';
+export const deleteDiskBtn = 'button[data-test-action="Delete"]';
+
+export enum disksTab {
+  addDisk = 'button[id="add-disk"]',
+  currVMStatus = 'span[data-test="resource-status"]',
```

Comment:
The name shouldn't be VMStatus, maybe diskStatus or resourceStatus?

Reply:
It's actually the VM status.

Reply:
then maybe change the name to `VMStatusLabel`?